README.md

```diff
@@ -116,57 +115,9 @@ Finally, properties are what give the selected elements of a rule declaration th
   - Uses of z-index
   - Compatibility or browser-specific hacks
 
-### OOCSS and BEM
+### Class Names
+TODO: How should we name classes?  My preference is something like `'section-element'` (ex. `'menu-block-title'`)
```

Comment:
I liked the BEM style of classes as it felt like a low cognitive load and less nesting overall